raiden_contracts/tests/test_channel_deposit.py

```diff
@@ -198,7 +198,7 @@ def test_channel_deposit_overflow(
     channel_identifier = create_channel(A, B)[0]
     channel_deposit(channel_identifier, A, deposit_A, B)
 
-    with pytest.raises(TransactionFailed):
+    with pytest.raises(TransactionFailed, match="abc"):
```

Comment:
What does "abc" refer to?

Reply:
Nice catch 👀 
I just used this as a message when replacing all contract invocations. As this test is ignored I've missed that.